src/gui/screen_printing_serial.cpp

```diff
@@ -22,25 +51,19 @@ typedef struct
 
     window_text_t w_message; //Messages from onStatusChanged()
     window_icon_t octo_icon;
-    window_icon_t tune_button;
-    window_text_t tune_label;
+
+    window_icon_t w_buttons[3];
```

Comment:
magical constants?

Reply:
I don't fully understand this comment - Added definition of buttons insted of fixed integers

Reply:
can we have something like
`static const uint32_t number_of_buttons = 3` and use it at appropriate places?

Reply:
also, it is a good practice to check at compile time that the size of w_buttons and w_labels is enough for all the items you want to put in. I assume you have something like:
`enum { BUTTON_TUNE=0, BUTTON_PAUSE=1, BUTTON_DISCONNECT=2 };`
One of the tricks is to add another item and document, that it must be the last one with highest number:
`... BUTTONS_COUNT=3`
and use the BUTTONS_COUNT for the array's sizes
or even better:
`static_assert( BUTTON_DISCONNECT < number_of_buttons );` later on